docs/samples/custom/kfserving-custom-model/kfserving_sdk_custom_image.ipynb

```diff
@@ -188,7 +188,10 @@
    "outputs": [],
    "source": [
     "%%bash --out CLUSTER_IP\n",
-    "echo \"$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')\""
+    "# Use \"kfserving-ingressgateway\" as your `INGRESS_GATEWAY` if you are deploying KFServing\n",
```

Comment:
Just for my own understanding, we will have a "kfserving-ingressgateway" as part of the kubeflow install?

Reply:
Yes "kfserving-ingressgateway" was added in this commit. https://github.com/kubeflow/manifests/commit/3f8c3582a7d18c390e5b691f10a0bee011c809d2
This PR is also related to this issue https://github.com/kubeflow/kfserving/issues/741